transports/ftp/src/main/java/org/mule/transport/ftp/FtpConnector.java

```diff
@@ -749,12 +752,22 @@ protected boolean validateFile(FTPFile file)
         return true;
     }
 
-    protected boolean isFile(ImmutableEndpoint endpoint, FTPClient client) throws IOException 
```

Comment:
Avoid changing method signature if possible. In this case, the signature change does not seem required to me. If it is needed, just add a new method and call it from the old one